app/models/pubmed_source_record.rb

```diff
@@ -29,12 +29,14 @@ def self.get_pubmed_hash_for_pmid(pmid)
   def self.get_pubmed_source_record_for_pmid(pmid)
     find_by(pmid: pmid) || get_pubmed_record_from_pubmed(pmid)
   end
+  private_class_method :get_pubmed_source_record_for_pmid
 
   # @return [PubmedSourceRecord] the recently downloaded pubmed_source_records data
   def self.get_pubmed_record_from_pubmed(pmid)
     get_and_store_records_from_pubmed([pmid])
     find_by(pmid: pmid)
   end
+  private_class_method :get_pubmed_record_from_pubmed
```

Comment:
This seems to be a candidate for a private method, i.e.
```
$ git grep 'get_pubmed_record_from_pubmed'  # on master
app/models/pubmed_source_record.rb:    find_by(pmid: pmid) || get_pubmed_record_from_pubmed(pmid)
app/models/pubmed_source_record.rb:  def self.get_pubmed_record_from_pubmed(pmid)
lib/tasks/cap_cutover.rake:      pubmed_source_record = PubmedSourceRecord.get_pubmed_record_from_pubmed(pmid)
spec/models/pubmed_source_record_spec.rb:  describe '.get_pubmed_record_from_pubmed' do
spec/models/pubmed_source_record_spec.rb:      record = described_class.get_pubmed_record_from_pubmed(pmid_created_1999)
spec/models/pubmed_source_record_spec.rb:      described_class.get_pubmed_record_from_pubmed(pmid_created_1999)
spec/models/pubmed_source_record_spec.rb:      record = described_class.get_pubmed_record_from_pubmed(pmid_created_1999)
spec/models/pubmed_source_record_spec.rb:        record = described_class.get_pubmed_record_from_pubmed(pmid)
```

Reply:
This PR proses to cull the `cap_cutover` tasks that use this public method.  If they can be removed, this method could be private.  It seems the wrapper in `get_pubmed_source_record_for_pmid` is a better public API that calls this method.